docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,217 @@
+---
+id: getting-started
+title: Getting Started on Linux
+layout: docs
+category: Quick Start
+permalink: docs/getting-started-on-linux.html
+next: android-setup
+---
+
+## Intro
```

Comment:
OK. Let me know if you want me to remove any of the casual language as well. I notice the rest of the docs are very straight and serious.